Week3/week3_homework/card_validator.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const test_valid_input = '6698666658666166';
+const test_invalid_input = 'a9224533219c0112';
+
+// #- Number must be 16 digits
+const digits_count = 16;
```

Comment:
Hi Hasan, I would like to refer you to the documentation for `Naming conventions` in JavaScript. I am not sure if this was discussed during the classroom but it will be very nice if you had a look at it - https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md

For constants like digits count, valid input and invalid input, it is correct that you used underscore to separate words but the convention is that the words should then be in UPPERCASE. Something like, DIGITS_COUNT, TEST_VALID_INPUT etc.

Also, while naming functions, first word of it should start with a verb which you do correctly with `validate`. But function names follow camelCasing by convention i.e `validateNumberOfDigits` etc.

Reply:
Thank you for this useful feedback, I'll stick to it from now on.